README.md

```diff
@@ -33,8 +33,8 @@ an enterprise. Inspired by Jupyter Kernel Gateway, Jupyter Enterprise Gateway pr
 
 - Adds support for remote kernels hosted throughout the enterprise where kernels can be launched in
   the following ways:
-  _ Local to the Enterprise Gateway server (today's Kernel Gateway behavior)
-  _ On specific nodes of the cluster utilizing a round-robin algorithm \* On nodes identified by an associated resource manager
+  \_ Local to the Enterprise Gateway server (today's Kernel Gateway behavior)
+  \_ On specific nodes of the cluster utilizing a round-robin algorithm * On nodes identified by an associated resource manager
```

Comment:
I think these underscores were intended to introduce indented bullets and they were meant to be hyphens.  Would you mind making that fix in this PR as well?
```suggestion
  - Local to the Enterprise Gateway server (today's Kernel Gateway behavior)
  - On specific nodes of the cluster utilizing a round-robin algorithm * On nodes identified by an associated resource manager
```